src/rpc/rawtransaction.cpp

```diff
@@ -1604,8 +1604,29 @@ UniValue joinpsbts(const JSONRPCRequest& request)
         merged_psbt.unknown.insert(psbt.unknown.begin(), psbt.unknown.end());
     }
 
+    // Generate list of shuffled indices
+    std::vector<int> input_indices(merged_psbt.inputs.size());
+    std::iota(input_indices.begin(), input_indices.end(), 0);
+    Shuffle(input_indices.begin(), input_indices.end(), FastRandomContext());
+    std::vector<int> output_indices(merged_psbt.outputs.size());
+    std::iota(output_indices.begin(), output_indices.end(), 0);
+    Shuffle(output_indices.begin(), output_indices.end(), FastRandomContext());
+
+    // Shuffle the inputs
```

Comment:
Unnecessary comment? Or maybe move it to L1607 like `Shuffle inputs and outputs of the merged PSBT`.

Reply:
Removed the comment and updated the one above.